x/leverage/keeper/math.go

```diff
@@ -14,3 +14,16 @@ func Interpolate(x, xMin, yMin, xMax, yMax sdk.Dec) sdk.Dec {
 	// y = y1 + m(x-x1)
 	return yMin.Add(x.Sub(xMin).Mul(slope))
 }
+
+// ApproxExponential is the taylor series expansion of e^x centered around x=0, truncated
+// to the cubic term. It can be used with great accuracy to determine e^x when x is very small.
+// Note that e^x = 1 + x/1! + x^2/2! + x^3 / 3! + ...
+func ApproxExponential(x sdk.Dec) sdk.Dec {
```

Comment:
Why truncated to the cubic term? 
![image](https://user-images.githubusercontent.com/17556614/170555197-c89823f4-8e7d-4902-af74-f46f00373190.png)
Like if we call ApproxExponential(2) it should return 6.33... instead of 7.38?